source/_integrations/ubus.markdown

```diff
@@ -57,7 +57,7 @@ Then, create an ACL file at `/usr/share/rpcd/acl.d/hass.json` for the user `hass
 }
 ```
 
-Restart the services. This ACL file needs to be recreated after updating/upgrading your OpenWrt firmware.
+Restart the services. Add the file to /etc/sysupgrade.conf so that it remains after updating/upgrading your OpenWrt firmware.
```

Comment:
```suggestion
Restart the services. Add the file path to /etc/sysupgrade.conf so that it remains after updating/upgrading your OpenWrt firmware.
```